src/sentry/tasks/auto_source_code_config.py

```diff
@@ -85,27 +85,37 @@ def process_error(error: ApiError, extra: dict[str, str]) -> None:
     default_retry_delay=60 * 10,
     max_retries=3,
 )
-def auto_source_code_config(project_id: int, event_id: str, **kwargs: Any) -> None:
+def auto_source_code_config(
+    project_id: int, group_id: int | None, event_id: str, **kwargs: Any
+) -> None:
     """
     Process errors for customers with source code management installed and calculate code mappings
     among other things.
 
     This task is queued at most once per hour per project.
     """
     project = Project.objects.get(id=project_id)
-    org: Organization = Organization.objects.get(id=project.organization_id)
+    org = Organization.objects.get(id=project.organization_id)
     set_tag("organization.slug", org.slug)
     # When you look at the performance page the user is a default column
     set_user({"username": org.slug})
     set_tag("project.slug", project.slug)
-    extra: dict[str, Any] = {"organization.slug": org.slug, "event_id": event_id}
-
-    event = eventstore.backend.get_event_by_id(project_id, event_id)
+    extra = {
+        "organization.slug": org.slug,
+        "project_id": project_id,
+        "group_id": group_id,
```

Comment:
do you have to access it using `**kwargs["group_id"]` too? or is making it optional in the signature enough?

Reply:
Let us not find out the wrong way.
I have changed it to `None` by default.

Reply:
It's fine to have it in the signature with `= None`